deps/v8/src/api.cc

```diff
@@ -3876,6 +3888,36 @@ void v8::RegExp::CheckCast(v8::Value* that) {
 }
 
 
+bool Value::BooleanValue() const {
+  return BooleanValue(Isolate::GetCurrent()->GetCurrentContext())
+      .FromMaybe(false);
```

Comment:
Depending on what behavior you prefer, you may want to use `FromJust()` here to cause a crash rather than failing silently.

Reply:
Done – this is currently only theoretical, right?

Reply:
Ah. Sorry. I actually meant all the other uses of FromMaybe, like in `Value::IntegerValue`. `BooleanValue` can actually never throw, so in newer V8 we restored this API already, without the `Maybe` return value.

Reply:
Okay, I understand – that would at least technically be a breaking change, I guess, and we’re past the cut-off date for that for Node 11